### DISCUSSION TITLE
Support MX record overrides

### DISCUSSION THREAD

**Discussion by andrew-cybsafe** [postalserver/postal#3155] [Feature suggestions]
It's helpful in certain scenarios to override the public MX records to route email directly. For example, MS suggests it as a good way to ensure email security or to bypass edge protections when sending large volumes of email. See: [MS | Use connectors to configure mail flow](https://learn.microsoft.com/en-us/exchange/mail-flow-best-practices/use-connectors-to-configure-mail-flow/use-connectors-to-configure-mail-flow#what-do-connectors-do).

To enable this, I propose that we add MX record overrides to postal, so that for the given domain postal will use the provided set of hostnames to send mail. Thoughts?

**Comment by willpower232** [comment]
I guess it depends on the audience for your email, if you were having some internal issues then maybe it would help but also the edge protections could come in handy if someone tries to send you a virus file or particularly scammy message through your website or whatever.

Presumably you could tell Postal to use the local resolver (`use_local_ns_for_domain_verification`) and have the alternate resolving records from some custom local resolver like dnsmasq.
